ui/admin-portal/src/app/components/candidates/find-candidate-source/find-candidate-source.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="input-group">
+  <ng-select
+    [items]="sources$ | async"
+    bindLabel="name"
+    [addTag]="true"
```

Comment:
I don't think you want [addTag] here - you can remove it as it is default to false. It is allowing users to add a tag if their auto search doesn't show anything - it won't do anything but it's redundant I think.
<img width="664" alt="Screen Shot 2024-10-14 at 3 52 05 pm" src="https://github.com/user-attachments/assets/589ea8e3-e73e-4665-96ed-483b9b61c8a5">

Reply:
Just reminder I think you want to remove the addTag input @camerojo - it's redundant but could be confusing

Reply:
Nice one! Thanks. I didn't know what addTag meant. Definitely don't want it.